pkg/parser/k8s/klog.go

```diff
@@ -18,15 +18,31 @@ import (
 	"regexp"
 	"strings"
 
-	"github.com/GoogleCloudPlatform/khi/pkg/log/schema"
+	"github.com/GoogleCloudPlatform/khi/pkg/model/enum"
 )
 
-var KLogSeverityFieldAlias = "@severity"
+// severityStringNotation maps string notation of severity found in logs to the severity types used in KHI.
+var severityStringNotation = map[string]enum.Severity{
+	"INFO":    enum.SeverityInfo,
+	"info":    enum.SeverityInfo,
```

Comment:
FYI: the lower letter(level=...) may be rely on here.
https://github.com/sirupsen/logrus/blob/v1.9.3/logrus.go#L59

ref: https://github.com/containerd/containerd/blob/v2.0.2/vendor/github.com/containerd/log/context.go#L71

Reply:
Then I'll add `panic` to be the fatal level on KHI. `trace` and `debug` to be default